modules/apps/translation/translation-test/src/testFunctional/tests/AutoTranslation.testcase

```diff
@@ -35,6 +35,443 @@ definition {
+
+		Translations.openToTranslateEntry(
+			groupName = "Site Name",
+			siteURLKey = "site-name",
+			sourceLanguageId = "en_US",
+			targetLanguageId = "es_ES",
+			webContentTitle = "This is the title of a web article in English");
+
+		Translations.viewTargetFields(
+			webContentContent = "This is the content of a web article in English",
+			webContentDescription = "Esta es la descripción de un artículo web en inglés",
+			webContentTitle = "This is the title of a web article in English");
+	}
+
+	@description = "This ensures that the numeric and decimal fields of a customized web content can be translated independently"
+	@priority = "5"
+	test CanTranslateNumericAndDecimalFieldsIndependently {
+		property test.name.skip.portal.instance = "AutoTranslation#CanTranslateNumericAndDecimalFieldsIndependently";
+
+		DMNavigator.openToConfigInSystemSettings(portlet = "Google Cloud Translator");
+
+		SystemSettings.configureSystemSetting(
+			enableSetting = "true",
+			settingFieldName = "Enabled",
+			settingName = "Service Account Private Key",
+			settingValue = PropsUtil.get("google.cloud.translation.service.key"));
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "WC Structure Description",
+			structureName = "WC Structure Name");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Numeric",
+			fieldName = "Numeric");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Decimal",
+			fieldName = "Numeric",
+			numericType = "Decimal");
+
+		WebContentStructures.saveCP(structureName = "WC Structure Name");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+		WebContent.addWithStructureCP(
+			numericType = "Decimal",
+			webContentNumeric = "1.0",
+			webContentNumericFieldLabel = "Decimal");
+
+		WebContent.addWithStructureCP(
+			webContentNumeric = "10",
+			webContentTitle = "This is the title of a web article in English");
+
+		PortletEntry.publish();
+
+		Translations.openToTranslateEntry(
+			groupName = "Site Name",
+			siteURLKey = "site-name",
+			sourceLanguageId = "en_US",
+			targetLanguageId = "es_ES",
+			webContentTitle = "This is the title of a web article in English");
+
+		Translations.translateSpecificFieldsWithGoogle(fieldNameList = "Decimal");
+
+		Translations.translateFields(
+			webContentDecimal = "1,0",
+			webContentNumber = "10");
+
+		Translations.translateSpecificFieldsWithGoogle(fieldNameList = "Numeric");
+
+		Translations.translateFields(
+			webContentDecimal = "1,0",
+			webContentNumber = "10");
+
+		PortletEntry.publish();
+
+		Translations.openToTranslateEntry(
+			groupName = "Site Name",
+			siteURLKey = "site-name",
+			sourceLanguageId = "en_US",
+			targetLanguageId = "es_ES",
+			webContentTitle = "This is the title of a web article in English");
+
+		Translations.translateFields(
```

Comment:
This should be viewing not translating

Reply:
@austinchiang The decimal 1.0 translated from `1.0` to `1,0 `, and numeric 10 can be translated successfully, but the value keeps with `10`

Reply:
Actually, there is a translation process when translating the numeric, just the value has not changed after translation.